test/LawsSpec.hs

```diff
@@ -18,3 +20,7 @@ spec = parallel $ do
     it "stateToIO should pass the laws" $
       property $ prop_lawfulState @'[Embed IO] $ fmap snd . stateToIO @Int 0
 
+  describe "History effects" $ do
+    it "runHistory should pass the laws" $
+      property $ prop_lawfulHistory $ runHistory @Int
+
```

Comment:
If we're going to add more laws for testing purposes, then I feel `LawsSpec` should be removed, and all laws should be placed in their respective effect's test spec.

Reply:
yeah, that's reasonable. the whole directory could use some cleaning love